src/senselab/audio/tasks/classification/speech_emotion_recognition.py

```diff
@@ -5,9 +5,8 @@
 
 from transformers import AutoConfig, pipeline
 
-from senselab.audio.data_structures.audio import Audio
-from senselab.utils.data_structures.device import DeviceType, _select_device_and_dtype
-from senselab.utils.data_structures.model import HFModel
+from senselab.audio.data_structures import Audio
+from senselab.utils.data_structures import DeviceType, HFModel, _select_device_and_dtype
```

Comment:
this almost seems less readable. not against it, just not sure about it

Reply:
> this almost seems less readable. not against it, just not sure about it

any idea on how we can make it better?